<launch>
    <!-- The following is denver UTM zone. check your UTM zone at  https://mangomap.com/robertyoung/maps/69585/what-utm-zone-am-i-in-# -->
    <param name="utm_zone" type="int" value="52" />
    <param name="hemisphere" type="string" value="North" />
    <node type="utm_lla_converter" pkg="utm_lla" name="utm_lla_converter" output="screen">
    </node>
	
<node type="utm_lla_converter" pkg="utm_lla" name="utm_lla_converter2" output="screen">
  <remap from="/ublox_gps/fix" to="/ublox_gps2/fix"/>
 <remap from="/utm" to="/utm2"/>
    </node>
</launch>
